whatshap/threading.py

```diff
@@ -191,7 +199,13 @@ def compute_cut_positions(path, block_cut_sensitivity, num_clusters):
             if dissim >= dissim_threshold:
                 cpn_rising = [False for c_id in range(num_clusters)]
                 cut_positions.append(i)
-    return cut_positions
+
+                # get all cut threads
+                threads_cut = [j for j in range(ploidy) if path[i - 1][j] in clusters_cut]
+                for thread in threads_cut:
+                    haploid_cut_positions[thread].append(i)
```

Comment:
Just as a suggestion for later: The code here is getting indented quite deeply. It would be nice to refactor this at some point.